File: slimadapter/src/main/java/com/jhj/slimadapter/listener/OnCustomLayoutListener.java
package com.jhj.slimadapter.listener;

import android.view.View;

import com.jhj.slimadapter.SlimAdapter;

/**
 * 自定义标题或尾部时自定义布局回调
 * <p>
 * Created by jhj on 18-10-11.
 */

public interface OnCustomLayoutListener<T extends SlimAdapter> {
    void onLayout(T adapter, View view);
}

File: slimadapterkt/src/main/java/com/zgdj/slimadapterkt/listener/OnCustomLayoutListener.kt
package com.zgdj.slimadapterkt.listener

import android.view.View

import com.zgdj.slimadapterkt.BaseAdapter


/**
 * 自定义标题或尾部时自定义布局回调
 *
 *
 * Created by jhj on 18-10-11.
 */

interface OnCustomLayoutListener<T : BaseAdapter<T>> {
    fun onLayout(adapter: T, view: View)
}
